daprdocs/content/en/developing-applications/authenticating/authenticating-aws.md

```diff
@@ -0,0 +1,50 @@
+---
+type: docs
+title: "AWS"
+linkTitle: "AWS"
+weight: 10
+description: "Information about authentication and configuration options for AWS"
+---
+
+All dapr components using various AWS services (DynamoDB, SQS, S3, etc) use a standardized set of attributes for configuration, these are described below.
+
+<https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials> provides a good overview of how the AWS SDK (which dapr uses) handles credentials
```

Comment:
Please use hyperlinks instead of links, e.g. in this case change to something like:
[This article](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials) provides a good overview ...

(same for other raw links below)